src/test/java/api/pet/Post_PositiveTest.java

```diff
@@ -0,0 +1,78 @@
+package api.pet;
+
+import api.domain.pets.Category;
+import api.domain.pets.PetStatus;
+import api.domain.pets.Tags;
+import api.utils.PetsUtils;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import api.apiwrappers.PetApiWrapper;
+import api.domain.pets.Pet;
+
+import java.util.ArrayList;
+
+public class Post_PositiveTest {
```

Comment:
Class name should be only camelcase, without underscore: `PostPositiveTest`. Also, we should create packages for positive and negative tests: e.g. `/java/api/positive/pet`

Reply:
Name changed to "PostPositiveTest".